### DISCUSSION TITLE
Distinguished names support

### DISCUSSION THREAD

**Discussion by krishnavatsavai** [polymind-inc/acmebot#729] [General]
Hi 

I am trying to create a cert using lets encrypt and wanted to use ACME bot with distinguished names. With specific values for CN,O, OU, L, S & C . Does the tool support it? Thank you.

**Comment by shibayan** [comment]
The current version does not support customization of the Subject, but Acmebot relies on the Issuance Policy of Key Vault, so if you edit it directly in the Azure Portal or elsewhere, that value should be used for the next certificate issuance.

![image](https://github.com/user-attachments/assets/d0aae17d-c899-45ed-ab4a-0cf6817bdc7b)
